satellite/metrics/counter.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package metrics
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/rpc/rpcstatus"
+	"storj.io/storj/satellite/metainfo"
+)
+
+// counter implements the metainfo loop observer interface for data science metrics collection.
+//
+// architecture: Observer
+type counter struct {
```

Comment:
I think we needed these counts per bucket.

Reply:
confirmed with @benjaminsirb it doesn't seem like we'll need counts per bucket